### PR TITLE
osx/*: add pt_BR translations

### PR DESCRIPTION
**Pull request by andreia**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
https://github.com/tldr-pages/tldr/blob/main/pages/osx/hdiutil.md
https://github.com/tldr-pages/tldr/blob/main/pages/osx/head.md
https://github.com/tldr-pages/tldr/blob/main/pages/osx/hidd.md
https://github.com/tldr-pages/tldr/blob/main/pages/osx/icalbuddy.md
https://github.com/tldr-pages/tldr/blob/main/pages/osx/imgcat.md
https://github.com/tldr-pages/tldr/blob/main/pages/osx/indent.md
https://github.com/tldr-pages/tldr/blob/main/pages/osx/internetsharing.md
https://github.com/tldr-pages/tldr/blob/main/pages/osx/istats.md

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.pt_BR/osx/head.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# head
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/osx/head.md:1]: Mandatory change: this page should be removed with the [original](https://github.com/tldr-pages/tldr/blob/main/pages/osx/head.md) one according our [specification](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#platform): commands available on more than one platform should be placed in `common/`.

File: pages.pt_BR/osx/hdiutil.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`hdiutil detach /Volumes/{{nome_do_volume}}`
+
+- Lista as imagens montadas:
+
+`hdiutil info`
+
+- Cria uma imagem ISO a partir do conteúdo de um diretório:
+
+`hdiutil makehybrid -o {{caminho/para/arquivo_de_saída}} {{caminho/para/diretório}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/osx/hdiutil.md:20]: Optional change: if `-o` long counterpart is available it can be used.

File: pages.pt_BR/osx/icalbuddy.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# icalBuddy
+
+> Utilitário de linha de comando para exibir eventos e tarefas do banco de dados do calendário do macOS.
+> Mais informações: <https://hasseg.org/icalBuddy/>.
+
+- Exibe eventos que acontecerão hoje:
+
+`icalBuddy -n eventsToday`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/osx/icalbuddy.md:8]:
```suggestion
`icalBuddy --includeOnlyEventsFromNowOn eventsToday`
```

Optional change (requires the [original](https://github.com/tldr-pages/tldr/blob/main/pages/osx/icalbuddy.md) page update): long option can be used.

File: pages.pt_BR/osx/icalbuddy.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Exibe tarefas incompletas:
+
+`icalBuddy uncompletedTasks`
+
+- Exibe uma lista formatada separada por calendário para todos os eventos de hoje:
+
+`icalBuddy -f -sc eventsToday`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/osx/icalbuddy.md:16]:
```suggestion
`icalBuddy --formatOutput --separateByCalendar eventsToday`
```

The same issue as above.

File: pages.pt_BR/osx/icalbuddy.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Exibe tarefas para um determinado número de dias:
+
+`icalBuddy -n "tasksDueBefore:today+{{dias}}"`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/osx/icalbuddy.md:20]:
```suggestion
`icalBuddy --includeOnlyEventsFromNowOn "tasksDueBefore:today+{{dias}}"`
```

The same issue as above.

File: pages.pt_BR/osx/indent.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# indent
+
+> Altera a aparência de um programa C/C++ inserindo ou excluindo espaços em branco.
+> Mais informações: <https://www.freebsd.org/cgi/man.cgi?query=indent>.
+
+- Formata código fonte C/C++ de acordo com o estilo Berkeley:
+
+`indent {{caminho/para/fonte.c}} {{caminho/para/fonte_identado.c}} -nbad -nbap -bc -br -c33 -cd33 -cdb -ce -ci4 -cli0 -di16 -fc1 -fcb -i4 -ip -l75 -lp -npcs -nprs -psl -sc -nsob -ts8`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/osx/indent.md:8]:
```suggestion
`indent {{caminho/para/fonte.c}} {{caminho/para/fonte_identado.c}} -nbad -nbap -bc -br -c33 -cd33 -cdb -ce -ci4 -cli0 -di16 -fc1 -fcb -i4 -ip -l75 -lp -npcs -nprs -psl -sc -nsob -ts8`
```

Warning (no fix available): too many short options can be converted to long ones.
Note: style guide should such case not to make commands too long after short->long option conversion.

File: pages.pt_BR/osx/indent.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+- Formata código fonte C/C++ de acordo com o estilo Kernighan & Ritchie (K&R):
+
+`indent {{caminho/para/fonte.c}} {{caminho/para/fonte_identado.c}} -nbad -bap -nbc -br -c33 -cd33 -ncdb -ce -ci4 -cli0 -cs -d0 -di1 -nfc1 -nfcb -i4 -nip -l75 -lp -npcs -nprs -npsl -nsc -nsob`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/osx/indent.md:12]:
```suggestion
`indent {{caminho/para/fonte.c}} {{caminho/para/fonte_identado.c}} -nbad -bap -nbc -br -c33 -cd33 -ncdb -ce -ci4 -cli0 -cs -d0 -di1 -nfc1 -nfcb -i4 -nip -l75 -lp -npcs -nprs -npsl -nsc -nsob`
```

The same issue as above.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Changes requested because at least one mandatory change found.
- Style guide doesn't handle some edge cases related to the current PR.
- Thanks for a contribution! ❤️

**Comment by andreia** [comment]
@EmilySeville7cfg Thank you for the review! :heart:
I've added the changes, please let me know if further changes are needed.

These are the PRs on original pages:
https://github.com/tldr-pages/tldr/pull/9533
https://github.com/tldr-pages/tldr/pull/9534

File: pages.pt_BR/osx/indent.md (hunk 6 above)
Comment (andreia) [reply pages.pt_BR/osx/indent.md:8]: Can we keep it this way? 
(I couldn't find the long counterpart)

Comment (EmilySeville7cfg) [reply pages.pt_BR/osx/indent.md:8]: Yes, we can because I can't enforce you to follow unwritten rules in our style guide. 😄

**Comment by EmilySeville7cfg** [comment]
@andreia, you can mark conversations as resolved if you think they are closed and not require any more comments. It will help me scan for present problems in PRs. :)

File: pages.pt_BR/osx/indent.md (hunk 7 above)
Comment (EmilySeville7cfg) [reply pages.pt_BR/osx/indent.md:12]: Marking as closed for the same reason as the conversation above.

**Review by EmilySeville7cfg — APPROVED** [review]
- No unresolved issues left, approved.
- Thanks for a contribution! ❤️

File: pages.pt_BR/osx/head.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages.pt_BR/osx/head.md]: Resolved in #9533.

File: pages.pt_BR/osx/icalbuddy.md (hunk 3 above)
Comment (EmilySeville7cfg) [reply pages.pt_BR/osx/icalbuddy.md]: Resolved in #9534.

File: pages.pt_BR/osx/icalbuddy.md (hunk 4 above)
Comment (EmilySeville7cfg) [reply pages.pt_BR/osx/icalbuddy.md]: Resolved in #9534.

File: pages.pt_BR/osx/hdiutil.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages.pt_BR/osx/hdiutil.md:20]: Unresolved because PR was merged.

**Pull request merged**